packages/text-annotator/src/SelectionHandler.ts

```diff
@@ -206,7 +209,7 @@ export const SelectionHandler = (
         if (selected.length !== 1 || selected[0].id !== hovered.id) {
           selection.userSelect(hovered.id, evt);
         }
-      } else if (!selection.isEmpty()) {
+      } else {
         selection.clear();
       }
```

Comment:
The `isEmpty` check is redundant because such a case is already handled on the a9s side. The operation will be no-op if the selection is already clear. [Source](https://github.com/annotorious/annotorious/blob/03ee23fa905a3fc448b4aa51d769c15ccb31c93a/packages/annotorious-core/src/state/Selection.ts#L45-L49)